1129/en.md

```diff
@@ -0,0 +1,107 @@
+# 1129 - Consistency Checker
+
+**Problem Link:** [click here](http://lightoj.com/volume_showproblem.php?problem=1129)
+
+**Algorithm:** [Trie Tree Data Structure on Shafaetsplanet](http://www.shafaetsplanet.com/?p=1679) **Shafaetsplanet** blog has really awesome tutorial on trie data structure. Also you can learn from anywhere.
+***Some youtube resources on trie:-***
+- [Tushar Roy - Coding Made Simple](https://m.youtube.com/watch?v=AXjmTQ8LEoI)
+- [HackerRank](https://m.youtube.com/watch?v=zIjfhVPRZCg)
+
```

Comment:
Either list down the resources/other tutorials to solve the problem in the end or try to combine them in your writing. I would prefer the latter. Here's a reference - https://github.com/lightoj-dev/problem-tutorials/pull/32/files#diff-94aeb337c51c48839ef777311d73c6d0fb848fcce5914df92d7f1e2973136b9cR7